aiida_quantumespresso/calculations/pp.py

```diff
@@ -216,9 +239,10 @@ def prepare_for_submission(self, folder):  # pylint: disable=too-many-branches,t
         calcinfo.codes_info = [codeinfo]
         calcinfo.local_copy_list = local_copy_list
         calcinfo.remote_copy_list = remote_copy_list
+        calcinfo.remote_symlink_list = remote_symlink_list
 
         # Retrieve by default the output file
-        calcinfo.retrieve_list = [self.inputs.metadata.options.output_filename]
+        calcinfo.retrieve_list = [self.inputs.metadata.options.output_filename, 'aiida.ref']
```

Comment:
I retrieve this because I may need to parse it. I could also add it to `retrieve_temporary_list`...